### PR TITLE
ci: use GH actions on 1.14 branch

### PR DESCRIPTION
**Pull request by nashif**
Move doc/license checks to use actions on 1.14 branch.

Signed-off-by: Anas Nashif <anas.nashif@intel.com>

### CODE REVIEW

**Comment by nashif** [comment]
@galak ping

File: west.yml [hunk 1]
```diff
@@ -34,7 +34,7 @@ manifest:
   # repositories.
   projects:
     - name: ci-tools
-      revision: cfb83fd6f1f498fa05d788a5d2cbeff9c92742f5
+      revision: pull/133/head
```
Comment (galak) [inline west.yml]: is this correct?  Pointing to a PR?

Comment (nashif) [reply west.yml]: yes, until https://github.com/zephyrproject-rtos/ci-tools/pull/133 is merged.

Comment (nashif) [reply west.yml]: now pointing to the merged commit in ci-tools

**Review by galak — APPROVED** [review]

**Review by carlescufi — APPROVED** [review]

**Pull request merged**